code/modules/mob/inventory.dm

```diff
@@ -467,30 +467,32 @@
 //any cost it has isn't a worry
 /mob/proc/change_number_of_hands(amt)
 	if(amt < held_items.len)
-		for(var/i in held_items.len to amt step -1)
+		for(var/i in held_items.len to amt+1 step -1)
```

Comment:
what?

Reply:
bug fix. this has been broken on tg for years.